scripts/style_checker.py

```
@@ -245,27 +240,78 @@ def analyze_parallelism(translations: List[str]) -> List[Dict[str, Any]]:
+    try:
+        response = client.chat.completions.create(
+            model="gpt-4o",
+            messages=[
+                {"role": "system", "content": "You are a linguistic analyst specializing in Japanese to English translation quality."},
+                {"role": "user", "content": prompt}
+            ],
+            response_format={"type": "json_object"},
+            temperature=0.0
+        )
+        return json.loads(response.choices[0].message.content)
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)